include/dxc/Support/dxcapi.use.h

```diff
@@ -28,7 +28,7 @@ class DxcDllSupport {
     ::dlclose(handle);
   }
   HMODULE LoadLibraryW(LPCWSTR name) {
-    return ::dlopen(CW2A(name).m_psz, RTLD_LAZY);
+    return ::dlopen(CW2A(name).m_psz, RTLD_NOW);
```

Comment:
Need some comments here to explain. Otherwise somebody later may think this is a potential optimization opportunity to change to `RTLD_LAZY`.

Reply:
Actually, I just double-checked, and adding the cmake fix is enough. This is not needed anymore. Fixed.